docs/VnVPlan/VnVPlan.tex

```diff
@@ -188,50 +189,138 @@ \subsection{Tests for Functional Requirements}
 \wss{Include a blurb here to explain why the subsections below
 	cover the requirements.  References to the SRS would be good here.}
 
-\subsubsection{Area of Testing1}
-
-\wss{It would be nice to have a blurb here to explain why the subsections below
-	cover the requirements.  References to the SRS would be good here.  If a section
-	covers tests for input constraints, you should reference the data constraints
-	table in the SRS.}
+\subsubsection{Work Orders}
 
-\paragraph{Title for Test}
+This section will contains tests covering the ``Work Orders'' requirements defined in the
+\href{https://github.com/arkinmodi/project-sayyara/blob/main/docs/SRS/SRS.pdf}{SRS}.
 
 \begin{enumerate}
 
-	\item{test-id1\\}
+	\item \textbf{FRT-BE17-1}
 
-	Control: Manual versus Automatic
+	      Control: Manual
 
-	Initial State:
+	      Initial State: Shop Owner/Employee Account with a registered shop and a quote
 
-	Input:
+	      Input: A new appointment created at the shop with a quote ID
 
-	Output: \wss{The expected result for the given inputs}
+	      Output: A new work order is created with details from the quote
 
-	Test Case Derivation: \wss{Justify the expected value given in the Output field}
+	      Test Case Derivation: The system shall automatically create a work order for new appointment with
+	      information from the quote
 
-	How test will be performed:
+	      How test will be performed: The tester will create an appointment through the user interface
 
-	\item{test-id2\\}
+	\item \textbf{FRT-BE17-2}
 
-	Control: Manual versus Automatic
+	      Control: Manual
 
-	Initial State:
+	      Initial State: Shop Owner/Employee Account with a registered shop
 
-	Input:
+	      Input: A new appointment created at the shop
 
-	Output: \wss{The expected result for the given inputs}
+	      Output: A new work order is created
 
-	Test Case Derivation: \wss{Justify the expected value given in the Output field}
+	      Test Case Derivation: The system shall automatically create a work order for new appointment
 
-	How test will be performed:
+	      How test will be performed: The tester will create an appointment through the user interface
 
-\end{enumerate}
+	\item \textbf{FRT-BE18}
 
-\subsubsection{Area of Testing2}
+	      Control: Manual
 
-...
+	      Initial State: Shop Owner/Employee Account with a registered shop, an upcoming appointment, and a
+	      work order associated to the appointment
+
+	      Input: Appointment is deleted by appointment ID
```

Comment:
Same here, will the system create the ID for appointments?

Reply:
This one is for deleting the appointment so we would need the appointment ID to know which on to delete. Since this is now manual I can reword it to say that it is deleted from the UI.